### PR TITLE
FIX: Updated Reusable cards

### PR DESCRIPTION
**Pull request by kotesh-arya**
- ## Cards were fixed to Reusability
 - ## Will reflect changes as shown in the Library

### CODE REVIEW

File: Components/Card/card.html [hunk 1]
```diff
@@ -265,22 +266,25 @@ <h2>Components</h2>
...
-                            Tony Stark is charismatic and eccentric, he is also very proud, but also very altruist
-                            and heroic. He did not have a good childhood, and this affects how he acts. Stark is
-                            very smart, and a very talented inventor.
+                            <p>TIMEX</p> <br />
+                            <div class="card-title">
+                                LEATHER WATCH SERIES
+                            </div>
+                            Rs.490 <strike>Rs.1299</strike> (62% OFF)
+                            <footer class="footer">
+                                <a href="">ADD TO CART</a>
```
Comment (vidhi-thakur) [inline Components/Card/card.html:280]: Just a suggestion: can make this a button instead of link

**Review by vidhi-thakur — APPROVED** [review]
the consistent use of `rem` and the code format looks good

File: Components/Card/card.css [hunk 2]
```diff
@@ -45,12 +47,44 @@ section .example-div {
...
+.card-image {
     width: 100%;
 }
-.horizontal-card-image{
+
+.horizontal-card {
+    display: flex;
+    width: 40rem;
+    margin-left: auto;
+    margin-right: auto;
```
Comment (NeerajVodala) [inline Components/Card/card.css]: margin: 0 auto; works the same (1 line code instead of 2)

File: Components/Card/card.css [hunk 3]
```diff
@@ -63,11 +97,22 @@ section .example-div {
     box-shadow: 3px 3px 20px 5px var(--black);
```
Comment (NeerajVodala) [inline Components/Card/card.css:96]: suggestion: make utility class for shadow so that you can reuse it

File: Components/Card/card.css [hunk 4]
```diff
@@ -63,11 +97,22 @@ section .example-div {
...
 }
 
-.text-overlay-card .overlay-title {
-    color: var(--white);
+.text-overlay-card {
     position: relative;
-    bottom: 4rem;
-    margin-bottom: -4rem;
+    height: 40rem;
+    box-shadow: 3px 3px 20px 5px var(--black);
```
Comment (NeerajVodala) [inline Components/Card/card.css:102]: same(utility class suggestion)

File: Components/Card/card.css [hunk 5]
```diff
@@ -63,11 +97,22 @@ section .example-div {
...
+.absolute-positioned-text {
+    color: var(--alert-error);
+    position: absolute;
+    z-index: 2;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    width: 25rem;
+    height: 100%;
+    background-color: rgba(197, 185, 185, 0.657);
```
Comment (NeerajVodala) [inline Components/Card/card.css:114]: use variables for colors if you are using it more than one time [Reason: you don't have to track and change its value everywhere instead could change at one place and see changes everywhere]

**Review by NeerajVodala — APPROVED** [review]
Great that you used relative units everywhere. Code looks good.
Left few suggestions.

File: Components/Card/card.css (hunk 2 above)
Comment (kotesh-arya) [reply Components/Card/card.css]: Sure will consider this and update 👍🏻

File: Components/Card/card.css (hunk 3 above)
Comment (kotesh-arya) [reply Components/Card/card.css:96]: Cool. Will add this 👍🏻

File: Components/Card/card.css (hunk 5 above)
Comment (kotesh-arya) [reply Components/Card/card.css:114]: Thanks for the suggestion, but I have used this only once. So kept it this way : )

**Pull request merged**